Models/Courses.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy.ext.declarative import declarative_base
+from  sqlalchemy import Column, Integer, String
+
+
+Base = declarative_base()
+
+
+class course(Base):
+    __tablename__ = 'courses'
+    department_ids = Column(String)
+    name = Column(String)
+    id = Column(Integer)
+    number = Column(Integer)
+
+    def __repr__(self):
+        return str(self.name)
```

Comment:
This isn't a very good representation -- at the very least, the `__repr__` should include the type of the object.